Feature/ci establishment (#16)
No-57/SCSH_3C_iOS: README.md

<img width="200" alt="Screenshot 2023-07-10 at 7 43 22 PM" src="https://github.com/No-57/SCSH_3C_iOS/assets/38306406/de6eb523-6b43-40ce-8bf5-afeb0c836aec">

## CI
### Prerequisites
- Install Fastlane: https://docs.fastlane.tools/getting-started/ios/setup/
- Set up Environment Variables: Environment variables are required by the CI script, Ensure that these are set up in your CI environment before running the script.

>   ``` console
>   // Create a file named `.env`
>   vi .env
> 
>   // Add all the variables you need.
>   // ...
> 
>   // export all variables on the terminal session.
>   source .env
>   ```

Reviewer: ditto

```suggestion
  ```sh
  # Create a file named `.env`
  $ vi .env

  # Add all the variables you need.
  # ...

  # export all variables on the terminal session.
  $ source .env
  ```
```

Author: [done](https://github.com/No-57/SCSH_3C_iOS/pull/16/commits/4d7ee5454c8023e37eb2f7eaa285901e52110070)

### Environment variables
1. `GIT_REPO_URL`: The URL of the Git repository.
2. `GIT_PROJECT_PATH`: The local directory path where the Git repo will be cloned.
3. `XCODE_PROJECT_PATH`: The path to the Xcode project file within the cloned repository.

> 

Reviewer: It would be great to add a whole `.env` example here.

Author: I simplify the script logic in order to reduce the redundant input parameter. 
please take a look again 🙏

### Execution

> - run the lane named 'ci_build' on platform 'ios'

Reviewer: Emphasize using **bold** or `backticks`.
```suggestion
- run the lane named **ci_build** on platform *ios*
```

or

```suggestion
- run the lane named `ci_build` on platform `ios`
```

Author: [done](https://github.com/No-57/SCSH_3C_iOS/pull/16/commits/4d7ee5454c8023e37eb2f7eaa285901e52110070)

>   ``` console
>   fastlane ios ci_build`

Reviewer: Use `sh` instead of `console`.

```suggestion
  ```sh
  $ fastlane ios ci_build
```

Author: [done](https://github.com/No-57/SCSH_3C_iOS/pull/16/commits/4d7ee5454c8023e37eb2f7eaa285901e52110070)
